pages/nft/iscn/_iscnId.vue

```diff
@@ -860,7 +864,17 @@ export default class NFTTestMintPage extends Vue {
       if (config) {
         messages.push(formatMsgCreateRoyaltyConfig(this.address, expectedClassId, config))
       }
-      await signingClient.sendMessages(this.address, messages);
+      const { mintMessages, sendMessages } = this.getNFTMessages(expectedClassId);
+      if (!this.isTransactionSizeLimited) {
+        messages = messages.concat(mintMessages);
+        messages = messages.concat(sendMessages);
```

Comment:
Can keep `message` to be `const` if using `push()`
```suggestion
        messages.push(...mintMessages, ...sendMessages);
```